modules/@apostrophecms/asset/index.js

```diff
@@ -11,17 +11,103 @@ module.exports = {
 
   options: {
     alias: 'asset',
+    // If true, create both modern JavaScript and ES5 (IE11) compatibility
+    // builds of the ui/src browser code in each module, and serve them
+    // to the appropriate browsers without overhead for modern browsers.
+    // This does not attempt to compile the admin UI (ui/apos) for ES5.
+    es5: false,
     // If this option is true and process.env.NODE_ENV is not `production`,
     // the browser will refresh when the Apostrophe application
     // restarts. A useful companion to `nodemon`.
-    refreshOnRestart: false
+    refreshOnRestart: false,
+    // Frontend builds. Changing these settings is not recommended
+    // except as part of development of the Apostrophe core. To enable
+    // the src-es5 build just set the `es5: true` option of this module.
+    builds: {
```

Comment:
The goal here isn't perfect configurability for future, unanticipated builds of every possible stripe right now. It's more to ensure we're taking an approach that can grow in that direction if needed, as requested.